client/my-sites/post-type-list/post-type-list-post-actions/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EllipsisMenu from 'components/ellipsis-menu';
+import PostTypeListPostActionsEdit from './edit';
+import PostTypeListPostActionsPublish from './publish';
+import PostTypeListPostActionsTrash from './trash';
+import PostTypeListPostActionsView from './view';
+import PostTypeListPostActionsRestore from './restore';
+
+export default function PostTypeListPostActions( { globalId } ) {
+	return (
+		<div className="post-type-list-post-actions">
+			<EllipsisMenu position="bottom left">
+				<PostTypeListPostActionsView globalId={ globalId } />
```

Comment:
>How would you handle custom actions?

I thought about this a bit. In terms of customization, do we want to enable a developer to replace the entire set of actions, or just append new custom options?

In either case, maybe this component could accept a prop `actions`, an array of menu item components, defaulting to the ones listed here, each being passed `globalId` of the post.